lib/process-services-cloud/src/lib/task/services/task-cloud.service.ts

```diff
@@ -189,6 +190,32 @@ export class TaskCloudService extends BaseCloudService {
         }
     }
 
+     /**
+      * Creates a new standalone task.
+      * @param taskDetails Details of the task to create
+      * @returns Details of the newly created task
+      */
+    createNewTask(taskDetails: StartTaskCloudRequestModel, appName: string): Observable<TaskDetailsCloudModel> {
```

Comment:
can you please change the property name from taskDetails startTaskRequest or payload